contentreview-impl/impl/src/java/org/sakaiproject/contentreview/impl/turnitin/TurnitinReviewServiceImpl.java

```diff
@@ -906,6 +926,156 @@ public void addTurnitinInstructor(Map userparams) throws SubmissionException, Tr
 	@SuppressWarnings("unchecked")
 	public void createAssignment(String siteId, String taskId, Map extraAsnnOpts) throws SubmissionException, TransientSubmissionException {
 
+		Site s = null;
+		try {
+			s = siteService.getSite(siteId);
+		} catch (IdUnusedException iue) {
+			log.warn("createAssignment: Site not found!" + iue.getMessage());
```

Comment:
It's much more useful to include the siteId you were searching for in the log, rather than relying on the message of the Exception not to change.